kuma/users/forms.py

```diff
@@ -87,13 +87,13 @@ class UserEditForm(forms.ModelForm):
             }
         ),
     )
-    mozillians_url = forms.CharField(
-        label=_("Mozillians"),
+    pmo_url = forms.CharField(
+        label=_("people.mozilla.org"),
```

Comment:
I think this should be:
```suggestion
        label=_("Mozilla People Directory"),
```